modules/sharing-registry/sharing-data-migrator/src/main/java/org/apache/airavata/sharing/registry/migrator/airavata/AiravataDataMigrator.java

```diff
@@ -181,6 +189,33 @@ public static void main(String[] args) throws SQLException, ClassNotFoundExcepti
             }
         }
 
+        //Creating application entries
+        query = "SELECT * FROM APPLICATION";
+        statement = expCatConnection.createStatement();
+        rs = statement.executeQuery(query);
+        while(rs.next()){
+            try {
+                Entity entity = new Entity();
+                entity.setEntityId(rs.getString("APPLICATION_ID"));
+                entity.setDomainId(rs.getString("GATEWAY_ID"));
+                entity.setEntityTypeId(rs.getString("GATEWAY_ID") + ":APPLICATION");
+                entity.setOwnerId(rs.getString("USER_NAME") + "@" + rs.getString("GATEWAY_ID"));
```

Comment:
Sneha, for the owner of an application deployment, have the migration script take an argument for a default owner that will be used to assign ownership in cases like this where there is no owner associated in the app catalog.

Reply:
Sure, Marcus. Do you have a text in mind for default owner?

Reply:
And what about the Entity's name?